### PR TITLE
Use lang attribute

### PR DESCRIPTION
**Pull request by Simounet**
Another quick one using the app language for the main html tag and the grabbed site one for our entry content.

### CODE REVIEW

**Comment by nicosomb** [comment]
You can maybe add a new test 😉

File: src/Wallabag/CoreBundle/Controller/EntryController.php [hunk 1]
```diff
@@ -270,6 +270,7 @@ public function redirectRandomEntryAction($type = 'all')
     public function viewAction(Entry $entry)
     {
         $this->checkUserAction($entry);
+        $entry->setLanguage($entry->getHTMLLanguage());
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Controller/EntryController.php]: I'm not a big fan of overriding the already saved data with something else only when rendering the entry.

Instead, you should keep your `getHTMLLanguage` and use it directly in the twig using `entry.html_language` (I'm not sure about the `_`)

File: src/Wallabag/CoreBundle/Entity/Entry.php [hunk 2]
```diff
@@ -787,6 +787,16 @@ public function getLanguage()
         return $this->language;
     }
 
+    public function getHTMLLanguage()
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Entity/Entry.php]: Provide a docblock to explain the purpose of that method

**Comment by Simounet** [comment]
Thanks for not letting me be a lazy boy. :D

**Comment by j0k3r** [comment]
You need to run the fixer locally: `php bin/php-cs-fixer fix`

**Comment by Simounet** [comment]
Done.

**Pull request merged**